packages/dart_frog_cli/lib/src/dev_server_runner/dev_server_runner.dart

```diff
@@ -258,12 +265,17 @@ class DevServerRunner {
       );
 
       logger.detail(
-        '''[process] dart $enableVmServiceFlag $serverDartFilePath''',
+        '''[process] dart $enableVmServiceFlag $serverDartFilePath ${arguments.join(' ')}''',
       );
 
       final process = _serverProcess = await _startProcess(
         'dart',
-        [enableVmServiceFlag, '--enable-asserts', serverDartFilePath],
+        [
+          enableVmServiceFlag,
+          '--enable-asserts',
+          ...arguments,
+          serverDartFilePath,
```

Comment:
I think this would be much stable, since the content of `...arguments` is injected and might lead to undesired effects when trailing the path.

```suggestion
          serverDartFilePath
          enableVmServiceFlag,
          '--enable-asserts',
          ...arguments,
```

Wdyt?